public/index.html

```diff
@@ -24,6 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://cdn.rawgit.com/adriancooney/console.image/c9e6d4fd/console.image.min.js"></script>
```

Comment:
> RawGit has reached the end of its useful life

Source: https://rawgit.com/

```suggestion
    <script src="https://raw.githubusercontent.com/adriancooney/console.image/master/console.image.min.js"></script>
```

Reply:
Looks like raw.githubusercontent is blocking requests.
Alternative:
https://www.jsdelivr.com/?docs=gh
Usage:
https://cdn.jsdelivr.net/gh/adriancooney/console.image/console.image.min.js